tests/missing/systemd/journal/__init__.py

```diff
@@ -21,6 +21,9 @@ class Reader(Iterator):
     def get_next(self):
         raise RuntimeError
 
+    def get_previous(self):
+        raise RuntimeError
```

Comment:
Could just make this return `{'__CURSOR': '0'}` since we don't test anything else.

Reply:
Will do.